runtime/compiler/env/ProcessorDetection.cpp

```diff
@@ -635,6 +635,14 @@ int32_t TR_J9VM::getCompInfo(char *processorName, int32_t stringLength)
             returnValue = snprintf(processorName, stringLength, "z14s (%d)", machineId);
          break;
 
+         case TR_Z15:
+            returnValue = snprintf(processorName, stringLength, "z15 (%d)", machineId);
+         break;
+
+         case TR_Z15s:
+            returnValue = snprintf(processorName, stringLength, "z15 (%d)", machineId);
```

Comment:
Should this string be `z15s` ?

Reply:
This was intentional. There is no `z15s`